src/main/java/org/springframework/data/couchbase/core/RxJavaCouchbaseOperations.java

```diff
@@ -53,6 +56,14 @@
 
     <T>Observable<T> remove(Iterable<T> batchToRemove, PersistTo persistTo, ReplicateTo replicateTo);
 
+    <T>Observable<T> cas(String id, Class<T> entityClass, Function<Optional<T>, CasResult<T>> updateFunction);
```

Comment:
The operations missing are insert, upsert and replace. Cas, asyncCas.. doesn't conform to the client library naming conventions